core/src/main/java/org/opensearch/remote/metadata/common/SdkClientUtils.java

```diff
@@ -436,7 +479,61 @@ private static NamedXContentRegistry createDefaultXContentRegistry() {
     }
 
     private static List<NamedXContentRegistry.Entry> getDefaultNamedXContents() {
-        Map<String, ContextParser<Object, ? extends Aggregation>> map = Map.ofEntries(
+        List<NamedXContentRegistry.Entry> entries = new ArrayList<>();
+
+        Map<String, ContextParser<Object, ? extends QueryBuilder>> queryMap = Map.ofEntries(
```

Comment:
This is applicable for all the clients I think? 

Does that mean local client search functionalities were missing more classes?

Reply:
> This is applicable for all the clients I think?

DDB Client doesn't have search, it delegates to this one.

> Does that mean local client search functionalities were missing more classes?

They were until we fixed it with #141.

These are only needed for parsing JSON received from somewhere else which only happens with the remote client.

Reply:
>These are only needed for parsing JSON received from somewhere else which only happens with the remote client.

I'm just wondering if there's any chance of breaking the local client existing functionality?

Reply:
> I'm just wondering if there's any chance of breaking the local client existing functionality?

This is only used in the `SdkClient.createParser()` util method.  That in itself is only used:
 - DDB Client for the simulated response parsing that doesn't actually need it
 - Remote Client for parsing JSON returned from the remote cluster
 - Various tests

Reply:
Thanks for the clarification.